examples/source_separation/conv_tasnet/README.md

```diff
@@ -13,15 +13,15 @@ For the usage, please checkout the [source separation README](../README.md).
 The default training/model configurations follow the best non-causal implementation from the paper. (causal configuration is not implemented.)
```

Comment:
IIRC, the following configurations are from paper setting. Since you are changing it, can you update the above statement as well?